smartseq2_single_sample/run_multi_samples/README.md

```diff
@@ -0,0 +1,46 @@
+# Run SmartSeq2 scRNA-Seq pipeline on Batch of Samples
+This workflow is for running SmartSeq RNA-Seq pipeline through batch of scRNA-Seq dataset.
+## Features
+ - Aligns reads of each sample to reference genome (using STAR)
+ - Performs quality control on generated BAM files (using Picard)
+   * [RNA metrics](https://broadinstitute.github.io/picard/picard-metric-definitions.html#RnaSeqMetrics)
+   * [Alignment metrics](https://broadinstitute.github.io/picard/picard-metric-definitions.html#AlignmentSummaryMetrics)
+   * [Duplication metrics](https://broadinstitute.github.io/picard/picard-metric-definitions.html#DuplicationMetrics)
+   * [Insertion Size metrics](https://broadinstitute.github.io/picard/picard-metric-definitions.html#InsertSizeMetrics)
+   * [RSEM transcrptome quantification](http://deweylab.biostat.wisc.edu/rsem/rsem-calculate-expression.html)
+   * [FeatureCounts gene/exon/transcript quantification](http://bioinf.wehi.edu.au/featureCounts/). FeatureCounts counts both unique mapping reads and multiple mapping reads.  
+
+# Getting Started
+## Download
+Use git clone `git clone git@github.com:HumanCellAtlas/skylab.git`
+## Requirements
+### Dockers
+- STAR(2_5_3a) docker: `humancellatlas/star_dev:v1`
+- RSEM docker: `humancellatlas/rsem`
+- Picard docker: `broadinstitute/genomes-in-the-cloud:2.3.1-1504795437`
+### Reference Genome And Gene Annotaiton
+ - Star reference indexes `star.tar`
+ - RSEM reference indexes `rsem.tar`
+ - Genome fasta
+ - refFlat file
+ - rRNA interval list
+ - Gencode annotation(gtf)
+## Input Json
+Main workflow `ss2_multi_sample_wf.wdl` takes a json file as input. An example json file is listed below:
+```json
+{
+  "Ss2RunMultiSample.ref_fasta": "File",
+  "Ss2RunMultiSample.rsem_genome": "String",
+  "Ss2RunMultiSample.ref_flat": "File",
+  "Ss2RunMultiSample.sra_dir": "String",
+  "Ss2RunMultiSample.star_genome": "String",
+  "Ss2RunMultiSample.sra_list_file": "File",
```

Comment:
Could you add more description for these, especially the inputs needed for the multiple workflow that aren't already required for the single sample one? I'm thinking of something like the descriptions you give for the cost script inputs in the scripts readme.

Reply:
sure, just added